src/main/java/muriplz/kryeittpplugin/KryeitTPPlugin.java

```diff
@@ -3,6 +3,7 @@
 import muriplz.kryeittpplugin.commands.*;
 import org.bukkit.Bukkit;
 import org.bukkit.ChatColor;
+import org.bukkit.command.TabCompleter;
```

Comment:
This has no use right? there is an option to optimize all imports from all classes

Reply:
Yeah it has no use Idk why it's there